mautic/mautic
# New contact's channel communication preference center

| Q | A |
| --- | --- |
| Bug fix? | n |
| New feature? | y |
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/137 |
| Related developer documentation PR URL |  |
| Issues addressed (#s or URLs) |  |
| BC breaks? |  |
| Deprecations? |  |
#### Description:

This PR adds new features to the existing frequency rules, and transforms it into a preference centre. Where your will have more control over how the contact will be contacted.
#### Steps to test this PR:
1. Apply this PR
2. if using sms make sure its enabled in the config
3. run migration
4. go to contact and see new features in frequency rules
